batch_update_users.py

```diff
@@ -41,24 +41,33 @@ def update_member(client, user_id, payload):
 
 
 def get_all_members(client):
-    try:
-        response = client.get(
-            'v2/members?includeArchived=true'
-        )
-        response.raise_for_status()
- 
-    except requests.exceptions.HTTPError as errh:
-        raise Exception(f"Connection Error: {errh}, API response: {errh.response.text}")
-    except requests.exceptions.ConnectionError as errc:
-        raise Exception(f"Connection Error: {errc}, API response: {errc.response.text}")
-    except requests.exceptions.Timeout as errt:
-        raise Exception(f"Timeout Error: {errt}, API response: {errt.response.text}")
-    except requests.exceptions.RequestException as err:
-        raise Exception(f"Other Error: {err}, API response: {err.response.text}")
-    else:
-        data = response.json()
-
-        return data
+    data = []
+    moreResults = True
+    nextPage = ''
+    while moreResults:
+        try:
+            response = client.get(
+                f'v2/members?includeArchived=true&limit=500{nextPage}'
+            )
+            response.raise_for_status()
+
+        except requests.exceptions.HTTPError as errh:
+            raise Exception(f"Connection Error: {errh}, API response: {errh.response.text}")
+        except requests.exceptions.ConnectionError as errc:
+            raise Exception(f"Connection Error: {errc}, API response: {errc.response.text}")
+        except requests.exceptions.Timeout as errt:
+            raise Exception(f"Timeout Error: {errt}, API response: {errt.response.text}")
+        except requests.exceptions.RequestException as err:
+            raise Exception(f"Other Error: {err}, API response: {err.response.text}")
+        else:
+            resp = response.json()
+            data = data + resp['entries']
+            if resp['nextPage'] is None:
+                moreResults = False
+            else:
+                pageID = str(resp['nextPage'])
+                nextPage = f'&page={pageID}'
```

Comment:
is any of this being used? I think we can just return resp['entries'] and maybe add a comment that it only returns 500 members